readme.md

```diff
@@ -3,15 +3,25 @@
 A tool to get notifications of the latest houses in your area with travel information to various locations.
 
 ## Setting up Dev environment
-- create a new file api_key.json in the root directory of the project and add your Zoopla API key into it, i.e.
+- create a new file credentials.json in the root directory of the project and add your Zoopla API key and gmail 
+credentials into it, i.e.
 
 ```
 {
-    "zoopla_api_key": "4adbe8c6dbe011e48ae714da"
+    "zoopla_api_key": "4adbe8c6dbe011e48ae714da",
+    "gmail_address": "something@gmail.com",
+    "gmail_password": "password"
 }
 ```
 
-- investigate config.py and update it with housing preferences relevant for you
+- investigate config.py and update it with housing preferences relevant for you (add emails of all the people who
+want to be notified when new houses are found in the house_member_email_addresses in JSON format, i.e
+```
+{
+    "email": "email@gmail.com",
+    "email2": "email2@gmail.com"
+}
+```
```

Comment:
The idea behind this was that you could keep track of whose email is which and maybe send emails with "Dear [name]". But I'm happy to change it to a list if that makes more sense.